rust/vapid/src/lib.rs

```diff
@@ -277,20 +289,17 @@ pub fn sign(key: Key, claims: &mut HashMap<String, serde_json::Value>) -> Result
 
     Ok(format!(
         "Authorization: {} t={},k={}",
-        SCHEMA,
-        auth_t,
-        auth_k
+        SCHEMA, auth_t, auth_k
     ))
 }
 
 pub fn verify(auth_token: String) -> Result<HashMap<String, serde_json::Value>, String> {
-
     //Verify that the auth token string matches for the verification token string
-    let auth_token = parse_auth_token(&mut String::from(auth_token.clone()))
+    let auth_token = parse_auth_token(&auth_token.clone())
```

Comment:
don't need to clone any longer

```suggestion
    let auth_token = parse_auth_token(&auth_token)
```